controllers/search.js

```diff
@@ -1,12 +1,7 @@
 'use strict';
 const querystring = require('querystring');
-const express = require('express');
 
-const { noStore } = require('../common/cached');
-
-const router = express.Router();
-
-router.get('/', noStore, (req, res) => {
+module.exports = function (req, res) {
     if (req.query.q) {
         const term = querystring.escape(req.query.q);
```

Comment:
While we're here, I wonder if we should log this somewhere (Datadog)? We don't have very good insight into this here as it's not tracked via GA and Google Search Console is a bit opaque sometimes.